### ISSUE TITLE
Feature Request: Allow chat messages from main conference to breakout rooms

### ISSUE THREAD

**Issue by tschzTA** [jitsi/jitsi-meet#14903]
### What problem are you trying to solve?

First off, I want to express my sincere appreciation for creating and maintaining such an awesome project.
Currently, it's not possible to send chat messages directly from the main Jitsi Meet conference to individual breakout rooms. This can be problematic when:

* Moderators need to announce the remaining time to all breakout groups.
* Moderators need to send important information or updates to specific groups.
* Participants want to communicate with someone in a different breakout room without leaving their own.

### What solution would you like to see?

Add a feature that allows users in the main conference (potentially just moderators) to select a breakout room and send a chat message that is visible only to the participants in that room.

This feature would greatly improve the flexibility and communication options within Jitsi Meet conferences that utilize breakout rooms. It would be particularly helpful for large-scale events and workshops.

### Is there an alternative?

While a full messaging system for all breakout room participants would be ideal, even the ability to directly message a single designated person in each breakout room would be a significant improvement. I understand breakout rooms operate in their own communication space, but enabling this basic form of direct messaging would streamline workflow and greatly enhance communication during large meetings.

**Comment by LittleBlueBreeze** [comment]
Yes, this is very much needed! This is kind of an essential feature when you're hosting breakout rooms, to be able to message everyone in them at once.

**Comment by tschzTA** [comment]
@LittleBlueBreeze We played around a bit and came up with a function that would loop the account through all breakout rooms, pasting the message in every room it visits. 
If it's really needed, you can do that, but of course, it's not a production-ready solution.

**Comment by LittleBlueBreeze** [comment]
Oh, interesting! Is there a guide somewhere on how to do that? I ended up doing that manually yesterday, but the other thing I needed to communicate with everyone was how long each breakout room had left. For that, my solution was to simply re-title each breakout room saying "1 minute left" or "30 seconds left" before closing them. Just mentioning this here in case it's useful to anyone else.
